lib/aac/utils.js

```diff
@@ -51,6 +51,8 @@ var isLikelyAacData = function(data) {
 
   return data.length >= offset + 2 &&
     (data[offset] & 0xFF) === 0xFF &&
+    // verify that the 2 layer bits are 0, aka this
```

Comment:
Should this comment be on the next line?

Reply:
done